### PR TITLE
BENCH-3686: Rename RStudio to R Analysis Environment

### PR DESCRIPTION
**Pull request by rogerwangcs**

### CODE REVIEW

File: src/r-analysis/docker-compose.yaml
Comment (yuhuyoyo) [inline src/r-analysis/docker-compose.yaml:9]: did you test this? i think you need to at least override the USER env var

Comment (rogerwangcs) [reply src/r-analysis/docker-compose.yaml:9]: nope, putting this up as a draft for now

Comment (rogerwangcs) [reply src/r-analysis/docker-compose.yaml]: The app creation works well now, I decided to leave the user + home directory as is since the rstudio app probably makes a lot of assumptions around that.

Not sure why the cirrocumulus test is failing though. It says the post startup is failing.

**Review by yuhuyoyo — APPROVED** [review]

File: src/r-analysis/docker-compose.yaml
Comment (yuhuyoyo) [reply src/r-analysis/docker-compose.yaml]: unlikely related to your change. if it's blocking you, can you file a ticket and disable cirro test?

**Comment by rogerwangcs** [comment]
Planning to merge this PR next week once the UI change lands next week. https://github.com/verily-src/terra-ui/pull/1847

**Comment by rogerwangcs** [comment]
Update: Instead of deleting the existing `rstudio` dir, I will leave it up and remove it separately once the UI renaming lands in prod.

**Pull request merged**
